### DISCUSSION TITLE
Custom FFMPEG commands

### DISCUSSION THREAD

**Discussion by ngysling** [mifi/lossless-cut#1640] [Ideas]
It would be great if there was a way to add custom FFMPEG commands to the output. It might already be there but having to export the files then manually reencode them so they encode properly is kind of a pain in the butt.

**Comment by mifi** [comment]
Can you provide a concrete example of what you want to do? which commands do you want to run, and when do you want to run them? why do you need to re-encode after export?

**Comment by ngysling** [comment]
Sure. I found with footage recorded with NVENC the output wouldn't embed in applications like Discord. I had to reencode it with x264 for it to work. I could record in x264 but it has a lot of CPU overhead.

**Reply by mifi** [reply]
So, which commands would you want to customise in losslesscut?

**Reply by Yahfz** [reply]
> So, which commands would you want to customise in losslesscut?

Since discord is so popular, would be good to have an option that detects what format/codec it is on and automatically re-encodes it to something discord embeds.

**Reply by mifi** [reply]
I think what you're asking for is https://github.com/mifi/lossless-cut/issues/372
